### PR TITLE
Iteration4

### PR DESCRIPTION
**Pull request by taylorjohnson141**
# Iteration Number
4
## Link to Iteration
- [x] Did you check all of the functionality of the webpage
- [x] Do both parties understand every line of code
- [x] Is the code DRY/SRP

## Questions

* Is using the local storage as the data model best practice /okay?

## Notes

* Need to refactor to make drier

### Pictures below show that the individual cards can now be deleted and favorited.

Screen Shot 2020-07-26 at 12 55 55 PM

Screen Shot 2020-07-26 at 12 56 30 PM

@kathrynljackson
@taylorjohnson141
@aemcdonald

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -319,7 +319,7 @@ background-color: white;
   }
 
 .idea-cards {
-  width: auto;
+  width: 100%;
```
Comment (kathrynljackson) [inline styles.css:322]: Fixed the grid layout responsiveness problem I was having yesterday!

**Comment by aemcdonald** [comment]
Code written together. I have understanding of functionality & of our process but want to revisit the code written today after our local storage/JSON lesson tomorrow to make sure that my understanding is solid.

**Review by aemcdonald — APPROVED** [review]

File: src/main.js [hunk 2]
```diff
@@ -9,19 +9,27 @@ var unfavoriteStar = document.querySelectorAll(".star-inactive");
 
 saveButton.disabled = true; //need to add class in CSS to change appearance
 saveButton.addEventListener("click", function(){
-  createIdeas();
-  displayIdeas();
+  var currentIdea = createIdeas();
+  displayIdeas(currentIdea);
   clearFields();
 });
+window.addEventListener('load', function(){
```
Comment (kathrynljackson) [inline src/main.js:16]: I think I understand the code here, but I want to discuss it together after tomorrow's class. I think I'll have a better understanding after that lesson and be able to use better terms of art with a more accurate explanation.

**Review by kathrynljackson — APPROVED** [review]
Coded together. Well discuss today's functions again after tomorrow's lesson.

**Comment by taylorjohnson141** [comment]
There are a couple of bugs that need fixed in the JS:
When a card is deleted and then another card is created a blank space appears.

**Pull request merged**